src/products/views/ProductUpdate/handlers.ts

```diff
@@ -236,6 +240,13 @@ export function createImageReorderHandler(
   };
 }
 
+function areVariantsEqual(
+  a: ProductDetails_product_variants,
+  b: ProductDetails_product_variants
+) {
+  return a.name === b.name;
```

Comment:
Why do you compare names, not ids? Theoretically, I guess, names might not be unique, while ids have to be.

Reply:
Good point